config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,17 @@
+require 'sidekiq/web'
```

Comment:
Basic configuration of sidekiq i wanted to handle the retries, but i judged it's outside the scope of the project

Reply:
There is a sidekiq setup where you can set the number of retries then it will just go to failure, i didn't bother with the configurations is what i'm trying to say here